cmd/krew/cmd/internal/read_plugin.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Kubernetes Authors.
```

Comment:
this cmd/ package does nearly nothing noteworthy that requires creating a new package for. especially because cmd/ directory directly has "tool" names (krew, generate-plugin-list, ...) this doesn't fit here.